public/index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Amira's Blog of Stuff</title>
+  </head>
+  <body>
+    <header>
+      <h1>Amira's Blog About Stuff</h1>
+
+      <nav>
+        <!-- <a href="#posts">Blog Posts</a> -->
+        <a href="./projects.html">Projects</a>
```

Comment:
See my above comment, re: routes and moving html files out of `/public`. I'd much rather the projects page be accesed via `<a href="/projects">`. How would you make that happen?